certified-connectors/BoldSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1536 @@
+            },
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "Location URL that can be used to unsubscribe the webhook"
+              }
+            }
+          }
+        },
+        "summary": "Webhook Events",
+        "description": "This triggers documents webhook event.",
+        "operationId": "WebHooks",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "events",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "items": {
+              "type": "string"
+            },
+            "description": "Select webhook event to which flow should be triggered.",
+            "x-ms-summary": "Select Webhook Events",
+            "x-ms-visibility": "important",
+            "enum": [
+              "Sent",
+              "Signed",
+              "Completed",
+              "Declined",
+              "Revoked",
+              "Reassigned",
+              "Expired",
+              "Viewed",
+              "AuthenticationFailed",
+              "DeliveryFailed",
+              "SendFailed",
+              "TemplateSendFailed",
+              "DraftCreated"
+            ]
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "name",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "Power Automate Webhook"
+                },
+                "events": {
+                  "type": "string",
+                  "description": "events",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "Sent"
+                },
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "x-ms-notification-url": true
+                },
+                "environment": {
+                  "type": "string",
+                  "description": "environment",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "Live"
+                },
+                "isActive": {
+                  "type": "boolean",
+                  "description": "isActive",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": true
+                },
+                "webhookType": {
+                  "type": "string",
+                  "description": "webhookType",
+                  "title": "",
```

Comment:
Please use proper title instead of empty value.

Reply:
Added proper title.